templates/sshd_config.erb

```diff
@@ -195,7 +197,9 @@ MaxSessions <%= @sshd_config_maxsessions %>
 <% end -%>
 
 #PermitTunnel no
-PermitTunnel <%= @sshd_config_permittunnel %>
+<% if @sshd_config_permittunnel_real =~ /(yes|no|point-to-point|ethernet)/ -%>
```

Comment:
Would `!= 'unset'` make this easier to change in the future as valid options to the parameter change?